packages/cli-platform-ios/src/tools/prompts.ts

```diff
@@ -35,14 +35,27 @@ export async function promptForConfigurationSelection(
 }
 
 export async function promptForDeviceSelection(
-  availableDevices: Device[],
+  devices: Device[],
+  lastUsedIOSDeviceId?: string,
 ): Promise<Device | undefined> {
+  const sortedDevices = [...devices];
```

Comment:
why are these devices sorted? I don't think they are...

Reply:
Not really sorted here, but I needed to create a value to do not mutate params.